xml/System.DirectoryServices.AccountManagement/PrincipalCollection.xml

```diff
@@ -604,7 +604,7 @@
       </ReturnValue>
       <Docs>
         <summary>Returns the count of <see cref="T:System.DirectoryServices.AccountManagement.Principal" /> objects in this collection.</summary>
-        <value>To be added.</value>
+        <value>The number of elements contained in the <see cref="T:System.Collections.Generic.ICollection`1" />.</value>
```

Comment:
NIT: might be better to say collection of `ICollection<Principal>` not `ICollection<T>` if possible, or something like
```suggestion
        <value>The number of <see cref="T:System.DirectoryServices.AccountManagement.Principal" /> contained in the collection.</value>
```